%% start of file `template.tex'.
%% Copyright 2006-2015 Xavier Danaux (xdanaux@gmail.com).
%
% This work may be distributed and/or modified under the
% conditions of the LaTeX Project Public License version 1.3c,
% available at http://www.latex-project.org/lppl/.


\documentclass[11pt,a4paper,sans]{moderncv}        % possible options include font size ('10pt', '11pt' and '12pt'), paper size ('a4paper', 'letterpaper', 'a5paper', 'legalpaper', 'executivepaper' and 'landscape') and font family ('sans' and 'roman')

% moderncv themes
\moderncvstyle{casual}                             % style options are 'casual' (default), 'classic', 'banking', 'oldstyle' and 'fancy'
\moderncvcolor{blue}                               % color options 'black', 'blue' (default), 'burgundy', 'green', 'grey', 'orange', 'purple' and 'red'
%\renewcommand{\familydefault}{\sfdefault}         % to set the default font; use '\sfdefault' for the default sans serif font, '\rmdefault' for the default roman one, or any tex font name
%\nopagenumbers{}                                  % uncomment to suppress automatic page numbering for CVs longer than one page

% character encoding
%\usepackage[utf8]{inputenc}                       % if you are not using xelatex ou lualatex, replace by the encoding you are using
%\usepackage{CJKutf8}                              % if you need to use CJK to typeset your resume in Chinese, Japanese or Korean

% adjust the page margins
\usepackage[scale=0.85, top=1.5cm]{geometry}
%\setlength{\hintscolumnwidth}{3cm}                % if you want to change the width of the column with the dates
%\setlength{\makecvheadnamewidth}{10cm}            % for the 'classic' style, if you want to force the width allocated to your name and avoid line breaks. be careful though, the length is normally calculated to avoid any overlap with your personal info; use this at your own typographical risks...

% personal data
\name{Govind}{Nair}
%\title{Resum√© title}                               % optional, remove / comment the line if not wanted
%\address{street and number}{postcode city}{country}% optional, remove / comment the line if not wanted; the "postcode city" and "country" arguments can be omitted or provided empty
\phone[mobile]{+1~(289)~933~4861}                   % optional, remove / comment the line if not wanted; the optional "type" of the phone can be "mobile" (default), "fixed" or "fax"
%\phone[fixed]{+2~(345)~678~901}
%\phone[fax]{+3~(456)~789~012}
\email{govind83nair@gmail.com}                               % optional, remove / comment the line if not wanted
%\homepage{www.johndoe.com}                         % optional, remove / comment the line if not wanted
\social[linkedin][www.linkedin.com/in/govind-nair-07b7aa4b]{GN}                        % optional, remove / comment the line if not wanted
%\social[xing]{john\_doe}                           % optional, remove / comment the line if not wanted
%\social[twitter]{jdoe}                             % optional, remove / comment the line if not wanted
\social[github][github.com/gogothegreen]{gogothegreen}                              % optional, remove / comment the line if not wanted
%\social[gitlab]{jdoe}                              % optional, remove / comment the line if not wanted
%\social[skype]{jdoe}                               % optional, remove / comment the line if not wanted
\social[researchgate][www.researchgate.net/profile/Govind_Nair5]{GN}
\social[googlescholar][scholar.google.com/citations?user=TvxdypsAAAAJ&hl=en]{GN}
%\extrainfo{additional information}                 % optional, remove / comment the line if not wanted
%\photo[64pt][0.4pt]{picture}                       % optional, remove / comment the line if not wanted; '64pt' is the height the picture must be resized to, 0.4pt is the thickness of the frame around it (put it to 0pt for no frame) and 'picture' is the name of the picture file
%\quote{Some quote}                                 % optional, remove / comment the line if not wanted

% bibliography adjustements (only useful if you make citations in your resume, or print a list of publications using BibTeX)
%   to show numerical labels in the bibliography (default is to show no labels)
%\makeatletter\renewcommand*{\bibliographyitemlabel}{\@biblabel{\arabic{enumiv}}}\makeatother
\renewcommand*{\bibliographyitemlabel}{[\arabic{enumiv}]}
%   to redefine the bibliography heading string ("Publications")
%\renewcommand{\refname}{Articles}

% Modifications n changes
% for reducing space in the signature at the end
\makeatletter
\renewcommand*{\makeletterclosing}{
  \@closing\\ [0.2em]
  {\bfseries\@firstname~\@lastname}%
  \ifthenelse{\isundefined{\@enclosure}}{}{%
    \\%
    \vfil%
    {\color{color2}\itshape\enclname: \@enclosure}}%
    \vfil}
\makeatother

% drawing the link symbol
\usepackage{tikz}
\newcommand{\ExternalLink}{%
    \tikz[x=1.2ex, y=1.2ex, baseline=-0.05ex]{% 
        \begin{scope}[x=1ex, y=1ex]
            \clip (-0.1,-0.1) 
                --++ (-0, 1.2) 
                --++ (0.6, 0) 
                --++ (0, -0.6) 
                --++ (0.6, 0) 
                --++ (0, -1);
            \path[draw, 
                line width = 0.5, 
                rounded corners=0.5] 
                (0,0) rectangle (1,1);
        \end{scope}
        \path[draw, line width = 0.5] (0.5, 0.5) 
            -- (1, 1);
        \path[draw, line width = 0.5] (0.6, 1) 
            -- (1, 1) -- (1, 0.6);
        }
    }
 
% social icon images for contact information    
\renewcommand*{\namefont}{\fontsize{28}{40}\mdseries\upshape}
\renewcommand*{\linkedinsocialsymbol}     {\includegraphics[width=0.5cm]{LI-In-Bug.png}~}
\renewcommand*{\githubsocialsymbol}       {\includegraphics[width=0.5cm]{Octocat.png}~}
\renewcommand*{\researchgatesocialsymbol} {\includegraphics[width=0.5cm]{researchgate_icon_130843.png}~}
\renewcommand*{\googlescholarsocialsymbol}{\includegraphics[width=0.5cm]{google-scholar-icon-1.png}~}

% bibliography with mutiple entries
%\usepackage{multibib}
%\newcites{book,misc}{{Books},{Others}}
%----------------------------------------------------------------------------------
%            content
%----------------------------------------------------------------------------------
\begin{document}
%\begin{CJK*}{UTF8}{gbsn}                          % to typeset your resume in Chinese using CJK
%-----       resume       ---------------------------------------------------------
\makecvtitle
\vspace*{-1cm}
\section{Education}
\cventry{2012--2017}{PhD, Computational Biology}{University of Natural Resources and Life Sciences (BOKU)}{Vienna}{Austria}{}  % Arguments not required can be left empty
\cventry{2010--2011}{MSc, Computational Biology}{University of East Anglia}{Norwich}{UK}{}
\cventry{2006--2007}{Postgraduate Diploma in Biotechniques}{Institute of Bioinformatics and Applied Biotechnology}{Bangalore}{India}{}
\cventry{2002--2006}{Bachelor of Engineering, Biotechnology}{Visveswaraiah Technological University}{India}{}{}

\section{Doctoral Thesis}

\cvitem{Title}{\emph{Metaheuristic Algorithms for Fast Prediction of Intervention Strategies in Metabolic Networks} \href{https://forschung.boku.ac.at/fis/suchen.hochschulschriften_info?sprache_in=en&menue_id_in=107&id_in=&hochschulschrift_id_in=11504}{\ExternalLink}}
\cvitem{Description}{Developing algorithms to predict intervention strategies that guarantee a high minimal yield for desired products in microbial systems using metaheuristic approaches like genetic algorithms and particle swarm optimization together with linear programming.}

\section{Experience}
\cventry{September 2021 --  present}{Freelance Editor}{Cactus Communications}{}{}{I am currently working part-time as a scientific editor for medical research  publications.}

\cventry{June 2017 -- May 2020}{Postdoctoral researcher}{University of Vienna}{Vienna}{Austria}{I worked on genome and metabolic evolution in Archaea. Here, I developed an algorithm to predict fusion proteins within thousands of genomes. Also developed an analysis pipeline to discover trends in evolution as well as gain insights into the metabolic capabilities of uncultured organisms.}

\cventry{May 2012 -- February 2015}{Junior Researcher}{Austrian Centre of Industrial Biotechnology (ACIB)}{Vienna}{Austria}{As a part of my PhD, I was employed at ACIB where I developed bioinformatic tools to rationally engineer microorganisms for improving production of industrially important chemicals. For this, I created two algorithms to solve the problem of finding optimal knockouts that ensure high product yield without compromising on organism growth.}

\cventry{November 2007 -- June 2009}{Scientist}{Abexome Biosciences}{Bangalore}{India}{Involved in various aspects of antigen production at a biotech company making diagnostic antibodies. Here I started out working as a molecular biologist in the wet-lab and later took on increasing responsibilities of collecting and analyzing data to facilitate laboratory work. Ended the stint in an operations role where I was responsible for getting the requirements of clients communicated effectively to the various teams within the company and ensure timely delivery of products.}

%------------------------------------------------
\subsection{Other Experience}

\cventry{March 2015 -- June 2015}{Marketing Head}{AprintaPro GmbH}{Vienna}{Austria}{During my PhD, I had the opportunity to be part of a 3D printing-related startup. I was involved in all aspects of the startup including marketing, product testing, meeting experts, attending startup focused events, contacting potential clients/partners and exploring options of raising capital.}

\cventry{January 2012 -- January 2013}{Co-founder}{Canvas88}{Bangalore}{India}{Two other people and I set up a company providing sales training. We developed a training program to help companies enhance the effectiveness of their sales teams. I was not involved in the day-to-day operations but and helped in getting clients and directing the growth strategy.}

\cventry{May 2011 -- August 2011}{Masters' Researcher}{The Genome Analysis Centre (TGAC)}{Norwich}{UK}{I worked here for my Master's project. TGAC is involved in the sequencing and analysis of genetic information from various sources. Here, I developed a pipeline to identify and classify viral DNA from metagenomic next generation sequencing data.}

\section{Publications}
\cvitem{}{$\bullet$ \textbf{Nair G}, Jungreuthmayer C, Zanghellini J (2017) Optimal knockout strategies in genome-scale metabolic networks using particle swarm optimization. - BMC Bioinformatics, 18:1:78.}
\cvitem{}{$\bullet$ Zanghellini J, Gerstl MP, Hanscho M,  \textbf{Nair G}, Regensburger G, M√ºller S, Jungreuthmayer C (2017) Toward Genome‚ÄêScale Metabolic Pathway Analysis. - Industrial Biotechnology: Microorganisms, 1:111-123.}
\cvitem{}{$\bullet$ \textbf{Nair G}, Jungreuthmayer C, Hanscho M, Zanghellini J (2015) Designing minimal microbial strains of desired functionality using a genetic algorithm. Algorithms for Molecular Biology, 10:29.}
\cvitem{}{$\bullet$ Jungreuthmayer C, \textbf{Nair G}, Klamt S, Zanghellini J (2013) Comparison and improvement of algorithms for computing minimal cut sets. BMC Bioinformatics 14:318.}

\section{Technical Skills}
%\subsection{Computer}
\cvitem{Languages}{Perl, Python (jupyter, pandas, NumPy, matplotlib), Bash, C, R, Java, MATLAB.}
\cvitem{Databases}{PostgreSQL}
\cvitem{Bioinformatics}{Algorithm development, optimization (CPLEX, GLPK), statistical analysis (R, numpy), modelling and analysis of metabolic networks (COBRA Toolbox). Bioinformatics databases like Uniprot, NCBI, STRING, KEGG, Pfam, etc,. Sequence analysis tools like BLAST, clustal omega, hidden Markov models, etc,. Visualization tools like Cytoscape.}
\cvitem{Machine learning}{Experience with scikit-learn and TensorFlow.}
\cvitem{General}{Slurm Workload Manager; Version control using Git and Subversion.}

\section{Languages}
\cvitem{}{English, German, +4 Indian languages.}

\section{Interests}
\cvitem{}{Rock climbing, reading, gardening, contact juggling, painting.}


\section{References}
\begin{cvcolumns}
  \cvcolumn{}{\begin{itemize}\item J√ºrgen Zanghellini (\textit{PhD Advisor})\item Christian Jungreuthmayer (\textit{ACIB Colleague})\item Filipa Souza (\textit{Postdoc PI})\end{itemize} \newline \vspace*{0.2cm} \textit{Contact details will be provided upon request.}}
  %\cvcolumn[0.5]{}{}
\end{cvcolumns}

% Publications from a BibTeX file without multibib
%  for numerical labels: \renewcommand{\bibliographyitemlabel}{\@biblabel{\arabic{enumiv}}}% CONSIDER MERGING WITH PREAMBLE PART
%  to redefine the heading string ("Publications"): \renewcommand{\refname}{Articles}
\nocite{*}
\bibliographystyle{plain}
\bibliography{publications}                        % 'publications' is the name of a BibTeX file

% Publications from a BibTeX file using the multibib package
%\section{Publications}
%\nocitebook{book1,book2}
%\bibliographystylebook{plain}
%\bibliographybook{publications}                   % 'publications' is the name of a BibTeX file
%\nocitemisc{misc1,misc2,misc3}
%\bibliographystylemisc{plain}
%\bibliographymisc{publications}                   % 'publications' is the name of a BibTeX file

\clearpage
%-----       letter       ---------------------------------------------------------
% recipient data
\recipient{Hiring Team}{BSI}
\date{\today}
\opening{Dear Sir or Madam,}
\closing{Sincerely,}
%\enclosure[]{}          % use an optional argument to use a string other than "Enclosure", or redefine \enclname
\makelettertitle

I am applying for the role of an Application Scientist at Bioinformatics Solutions Inc. I have a PhD in computational biology followed by a three-year stint as a postdoctoral researcher. I also have worked in the biotech industry for two years prior to pursuing my graduate studies. After reviewing the job requirements, I believe that my decade-long experience in developing bioinformatics algorithms and pipelines, and wet- ab experience make me a good fit for this position.
 
I bring with me a wide range of experience in bioinformatics. My bioinformatics journey began at my first job, where I started in the wet lab, performing basic molecular biology work like DNA and RNA extraction, PCR, cloning and expression of recombinant proteins, and protein purification using SDS-PAGE. Later I took on increasing bioinformatics-related responsibilities, using tools and databases like NCBI, Pubmed, and PDB to support the planning of experiments, particularly, primer design and antigen selection. To strengthen my bioinformatics programming skills, I did a masters in computational biology. In addition to gaining basic skills in Java, Matlab, SQL, and Perl, I gained significant practical experience during this time by working at TGAC (currently the Earlham Institute), where I developed a pipeline to analyze viral metagenomic NGS data. Using Perl I integrated and adapted existing tools for bacterial metagenomic analysis. Realizing the importance of networks in biology, I moved into metabolic network analysis for my PhD. Coding mostly in Perl and Matlab, along with use of optimization tools like CPLEX and GLPK, I translated mathematical ideas into practical algorithms for strain improvement, developing two new, state of the art algorithms. For my postdoc, I joined a group working on the origin of life problem, particularly the origin of metabolism. Here I developed a method to find and analyze fusion proteins in prokaryotic genomes, which involved computational analysis of a functionally wide range of proteins in thousands of organisms. Here too, I coded mostly in Perl but also used R for statistical analysis and visualizations, integrating data from a range of databases like Uniprot, NCBI, STRING, KEGG, Pfam, etc., and using tools like BLAST, clustal omega, hidden markov models, etc., to assign functions to proteins. I have worked mostly on UNIX systems and my programs were either multicore or ran on high-performance clusters.

I have the necessary programming skills, mathematical, statistical, and biological knowledge, and experience to take on challenges across a wide range of bioinformatics problems. This includes the capability to deliver on all of the responsibilities mentioned in the job posting. I also have a background in engineering, with a solid foundation in mathematics, statistics, and modelling, which I have used throughout my research career for better insights into problems. I have had to learn new programming languages, tools, and resources in every job I have held so far. I took these as learning opportunities and always delivered on my responsibilities. Similarly, at BSI, I will be quickly up to speed with your tools and techniques. Although I have no experience with computer software for LC-MS/MS analysis, I have a theoretical understanding of these tools and have worked with people using them. I enjoy working with data and am always eager to learn new skills and technologies. Particularly, I have experience of generating biological insights from data, integrating sequence information, reaction and pathway databases, and published literature. I am self-taught in deep learning and have implemented various neural network architectures using TensorFlow. I also like working with people; for me, communicating with people having different knowledge and skills than my own is a learning opportunity. As I was working in an academic setting before, I have regularly presented my work at conferences to diverse groups of people. I have also helped organize talks, conferences, and social events. I believe that I will be able to draw on my considerable experience as a bioinformatics researcher to support the bioinformatics needs at BSI.

I moved to Canada in July 2020 and have a permanent residency here. Please do reach out to me if you need more information about my profile. I look forward to an opportunity to speak with you and discuss the position in more detail. Thank you for your time and consideration.


\vspace*{0.3cm}


\makeletterclosing
\name{}{} %this is needed to suppress latex from printing name in the footer, causing a repetition as name is also printed by \makeletterclosing

%\clearpage\end{CJK*}                              % if you are typesetting your resume in Chinese using CJK; the \clearpage is required for fancyhdr to work correctly with CJK, though it kills the page numbering by making \lastpage undefined
\end{document}

